yarn.lock

```diff
@@ -10447,7 +10438,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"classnames@npm:2.3.1":
+"classnames@npm:^2.2.5":
```

Comment:
For information, `classnames` is still there because the demo needs `recharts` which uses it. It won't be installed by react-admin users unless they use a library depending on it

Reply:
Yep, I so it